pymatviz/structure_viz.py

```diff
@@ -410,8 +410,10 @@ class used to draw chemical bonds. Allowed are edgecolor, facecolor, color,
         ):
             try:
                 struct.add_oxidation_state_by_guess()
-            except ValueError:  # fails for disordered structures
-                "Charge balance analysis requires integer values in Composition"
+            except ValueError as exc:  # fails for disordered structures
+                raise ValueError(
+                    "Charge balance analysis requires integer values in Composition"
```

Comment:
This change leads to multiple failures in unit tests, I'm not sure what is intended for this part? Suppress the `ValueError` or re-raise an `ValueError` with custom message?